actions/promote_package.yaml

```diff
@@ -5,6 +5,9 @@ description: Promote a package to production from staging.
 enabled: true
 entry_point: workflows/promote_package.yaml
 parameters:
+  token:
+    type: string
+    description: Packagecloud API token
```

Comment:
Can you set default value to get from the datastore? i.e. {{system.pkg_cloud_token}}